bedrock/mozorg/templates/mozorg/moss/index-092018.html

```diff
@@ -77,24 +76,14 @@ <h2 class="moss-section-heading">{{ _('Award Selection') }}</h2>
     {{ _('and') }}
     <a href="https://www.justindorfman.com/">Justin Dorfman</a>.
   </p>
-
-  <p class="moss-copy">
-    {{ _('The Track IV Global Mission Partners: India Committee members are') }}
-    <a href="https://twitter.com/MishiChoudhary">Mishi Choudhary</a>,
-    <a href="https://www.linkedin.com/in/sushantomitra/">Sushanto Mitra</a>,
-    <a href="https://wikipedia.org/wiki/Nagarjuna_G.">Nagarjuna Gadiraju</a>,
-    <a href="http://alolitasharma.com/">Alolita Sharma</a>
-    {{ _('and') }}
-    <a href="http://www.cmu.edu/iii/innovators/faculty-staff/wasserman.html">Tony Wasserman</a>.
-  </p>
 </section>
 
 <section class="moss-section">
   <h2 class="moss-section-heading">{{ _('Past Recipients') }}</h2>
 
   <p class="moss-copy">
   {% trans %}
-    Past MOSS awardees include:
+    Past MOSS awardees include the projects listed below. For more information on past MOSS awards, please see the <a href="https://github.com/mozilla/MOSS-Awards-Directory">MOSS awards directory</a> on GitHub.
```

Comment:
Please pass `href` values as variables via the `{% trans %}` block, like we do for other copy blocks. Link destinations should not be part of the translatable strings.